src/main/java/mate/academy/spring/util/EmailValidator.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.spring.util;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailValidator {
+    private static final String PATTERN = "\\b[A-Z0-9._%-]+@[A-Z0-9.-]+\\.[A-Z]{2,4}\\b";
+
+    public boolean isValid(String email) {
+        return email.matches(PATTERN);
+    }
+}
```

Comment:
in this task u need to create ur annotation for validating emails, so lets do it